hydro_deploy/hydroflow_plus_deploy/CHANGELOG.md

```diff
@@ -81,11 +81,11 @@
 
  - <csr-id-7f68ebf2a23e8e73719229a6f0408bffc7fbe7af/> simplify Location trait to remove lifetimes
  - <csr-id-77f3e5afb9e276d1d6c643574ebac75ed0003939/> simplify lifetime bounds for processes and clusters
-   feat(hydroflow_plus): simplify lifetime bounds for processes and
-   clusters
-   
-   This allows `extract` to move the flow builder, which is a prerequisite
-   for having developers run the optimizer during deployment as well in
+   feat(hydroflow_plus): simplify lifetime bounds for processes and
+   clusters
+   
+   This allows `extract` to move the flow builder, which is a prerequisite
+   for having developers run the optimizer during deployment as well in
    case it changes the network topology.
```

Comment:
can you double check that these are not CRLF line ends?

Reply:
Seems to be okay, this is what is generated by `cargo changelog`